README.md

```diff
@@ -107,7 +107,6 @@ To run tests, there are a few cargo make tasks we can run:
 - `test`: tests the whole cargo workspace and ETH contracts. Requires a `--profile` argument.
 - `test-workspace`: tests only the cargo workspace.
 - `test-contracts`: tests only the contracts.
-- `test`: tests the whole cargo workspace, ETH contracts and runs modexp benchmarks. Requires a `--profile` argument.
```

Comment:
I guess it's better to leave this kind of description, because it includes `modexp benchmarks`.

Reply:
fixed